src/registrar/models/user_portfolio_permission.py

```diff
@@ -99,20 +99,26 @@ def clean(self):
         """Extends clean method to perform additional validation, which can raise errors in django admin."""
         super().clean()
 
-        # Check if a user is set without accessing the related object.
-        has_user = bool(self.user_id)
-        if self.pk is None and has_user:
-            existing_permissions = UserPortfolioPermission.objects.filter(user=self.user)
-            if not flag_is_active_for_user(self.user, "multiple_portfolios") and existing_permissions.exists():
-                raise ValidationError(
-                    "This user is already assigned to a portfolio. "
-                    "Based on current waffle flag settings, users cannot be assigned to multiple portfolios."
-                )
-
         # Check if portfolio is set without accessing the related object.
         has_portfolio = bool(self.portfolio_id)
         if not has_portfolio and self._get_portfolio_permissions():
             raise ValidationError("When portfolio roles or additional permissions are assigned, portfolio is required.")
 
         if has_portfolio and not self._get_portfolio_permissions():
             raise ValidationError("When portfolio is assigned, portfolio roles or additional permissions are required.")
+
+        # Check if a user is set without accessing the related object.
```

Comment:
clarifying question: what does not accessing the related object entail?

Reply:
It means we don't actually check if an object with the given user id exists...we just check that a user id is present.

Reply:
Thank you for clarifying! Is there a reason we specify here that we check the user id without checking for the object itself? I know it's from existing code but I noticed these comments' wording make it confusing on what they're trying to communicate (ie whether it's relevant that we're looking up the id without the object) so left a suggestion to make the comment more concise. You have more exposure to this model, so feel free to ignore the suggestion if you think it's necessary though!